docs/CONTRIBUTING.md

```diff
@@ -12,22 +12,70 @@ request.
 
 It is important to note that there are **two types of PRs** for this repository:
 
-1. **New Userstyles:** Please see [userstyle-creation.md](userstyle-creation.md).
-2. **Other improvements:** These should follow the recommendations below, to help
-   improve the chances of a merged PR.
+1. **New Userstyles:** Please see
+   [userstyle-creation.md](./userstyle-creation.md).
+2. **Other improvements:** These should follow the recommendations below, to
+   help improve the chances of a merged PR.
+## Recommendations
 
-## General Recommendations
+### Root Variables
+
+Whilst writing an userstyle, its likely you have come across something like
+`--black: #000;` in the `:root` selector. These are called root variables, and
+they are used to store values that can be reused throughout the userstyle. In
+userstyles, its important that we prefer using these over theming individual
+elements, as it allows for a more consistent and maintainable userstyle.
+
+### Opinionated Changes
+
+When making changes to the userstyle, it is important to remember that different
+users would like to have different things. This is why we have a set of rules
+that we follow when making changes to the userstyle. This is why we have the
+linter to flag any changes that do not follow these rules. These rules include:
+
+- No font changes
+- No layout changes
+  - Padding, margin, and width changes
+  - Hiding elements
+  - Changes to the position of elements
+
+
+### General Recommendations
 
 - Create a
   [topic branch](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows#_topic_branch)
   on your fork for your specific PR.
-- Catppuccin uses the [Conventional Commits
-  ](https://www.conventionalcommits.org/en/v1.0.0/) standard for creating explicit and meaningful commit messages. This repository requires pull request _titles_ to be in the conventional commit format, however we do not require it for individual commits within a pull request.
-- Update the version in the `==UserStyle==` header of the `catppuccin.user.css` file. This is to enable version control of the style.
+- Catppuccin uses the
+  [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+  standard for creating explicit and meaningful commit messages. This repository
+  requires pull request _titles_ to be in the conventional commit format,
+  however we do not require it for individual commits within a pull request.
+- Update the version in the `==UserStyle==` header of the `catppuccin.user.css`
+  file. This is to enable version control of the style.
```

Comment:
More formatting changes... I think this is `deno fmt` at work creating these line breaks. Not a fan personally, I'll leave it up to you and Hammy though if we mind or not.

Reply:
I honestly don't mind, not like we see the markdown compared to the rich display.

Reply:
Eeh I mean it makes it easier to edit...

Reply:
I think prettier is going to change these anyways? I think I've seen it running over markdown :thinking:

Reply:
> I think prettier is going to change these anyways? I think I've seen it running over markdown 🤔
> 
> Edit: Looks like it -> [b5cbfeb](https://github.com/catppuccin/userstyles/commit/b5cbfeb8a0e4d01963d7fe6a6daae035981413ce)

No, it doesn't remove forced line breaks.